packages/flutter/lib/src/material/text_field.dart

```diff
@@ -781,6 +782,9 @@ class TextField extends StatefulWidget {
   /// {@macro flutter.widgets.undoHistory.controller}
   final UndoHistoryController? undoController;
 
+  /// Determine if onTapOutside can be called when the [TextField] is not focused.
```

Comment:
nit: `Determine whether.....`. This follows similar documentation for bools in this file.